src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -8553,6 +8559,59 @@ private static boolean handleGhostOfCommerce(String text, TagStatus status) {
     return false;
   }
 
+  private static final Pattern[] COOKBOOKBAT_QUEST = {
+    Pattern.compile(
+        "\"As I recall, .*? was common in (?<location>.*?), back in my day\\. +Perhaps if you kill an? (?<monster>.*?), you'll find one\\.\""),
```

Comment:
would be good to run with this a bit (I assume everybody will complain to garbo if it doesn't work), as I think pronouns can be more than just a/an: other places have a/an/the/some/(blank)

I don't know whether specifically CBB ingredient-monsters have such a variety

Reply:
also would be nice to check that each of the messages matches, as I think we had that problem before with Robortender.